src/main/java/org/jenkinsci/plugins/vSphereCloudProvisionedSlave.java

```diff
@@ -57,13 +57,20 @@ public vSphereCloudProvisionedSlave(String name, String nodeDescription,
     @Override
     protected void _terminate(TaskListener listener) throws IOException, InterruptedException {
         super._terminate(listener);
-        final vSphereCloudLauncher launcher = (vSphereCloudLauncher) getLauncher();
-        if (launcher != null) {
-            final vSphereCloud cloud = launcher.findOurVsInstance();
-            if (cloud != null) {
```

Comment:
I managed to miss off the extra Javadoc I'd added to make this clear from my PR.
I've added another commit that adds this.
(my local code had this javadoc already, but I'm in the process of un-picking my changes into a series of discrete PRs, and this comment got missed out.  Sorry for the confusion)